packages/plugin/src/theia.d.ts

```diff
@@ -25,7 +25,7 @@ import './theia-proposed';
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable max-len */
 
-export module '@theia/plugin' {
+declare module '@theia/plugin' {
```

Comment:
This update unfortunately breaks our typedocs generation for `@theia/plugin` (fixed by https://github.com/eclipse-theia/theia/pull/10274). We may need to revisit the typedocs fix since it now produces a report with these changes like:

![typedocs](https://user-images.githubusercontent.com/40359487/187430402-3a97c175-b049-4172-97f3-1d3bb380f432.png)